<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSerialPort</class>
 <widget class="QDialog" name="ConfigSerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>349</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbSerialPort">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>91</width>
     <height>105</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBaudRate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>87</y>
     <width>80</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>BaudRate</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbDataBits">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>80</width>
     <height>105</height>
    </rect>
   </property>
   <property name="text">
    <string>DataBits</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbParity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>187</y>
     <width>80</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSerialPort">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbBaudRate">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbDataBits">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbParity">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbFlowControl">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>285</y>
     <width>111</width>
     <height>49</height>
    </rect>
   </property>
   <property name="text">
    <string>FlowControl</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbStopBits">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbStopBits">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>80</width>
     <height>105</height>
    </rect>
   </property>
   <property name="text">
    <string>StopBits</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFlowControl">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigSerialPort</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigSerialPort</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
